app/tests/system/nightwatch-config.js

```diff
@@ -21,35 +22,40 @@ module.exports = {
     'test_settings': {
         'default': {
             'globals' : {
-                'waitForConditionTimeout' : 60000
+                'waitForConditionTimeout' : 60000,
+                'waitForConditionPollInterval': 500
             },
+            'end_session_on_fail': false,
             'launch_url': 'http://localhost:4723/wd/hub',
             'selenium_host': 'localhost',
             'selenium_port': 4723,
             'silent': true,
             'output': true,
             'screenshots': {
-                'enabled': false,
-                'path': ''
+                'enabled': true,
+                'path': '${CIRCLE_ARTIFACTS}',
+                'on_failure': true
             },
             'desiredCapabilities': {
-                'fullReset': false,
                 'app': appPath,
                 'platformName': 'iOS',
-                'platformVersion': '8.4',
+                'platformVersion': '9.2',
                 'deviceName': 'iPhone 6'
             },
-            'exclude': 'nightwatch-config.js'
+            'exclude': ['nightwatch-config.js', 'pageObjects', 'assertions']
         },
 
-        'ios-sim': {
+        'android': {
             'desiredCapabilities': {
-                'fullReset': false,
```

Comment:
`fullReset` is `false` by default. Reference: http://appium.io/slate/en/master/?ruby#appium-server-capabilities